com.unity.ml-agents/Runtime/Sensors/CameraSensorComponent.cs

```diff
@@ -75,6 +75,11 @@ public bool Grayscale
             set { m_Grayscale = value; }
         }
 
+        [HideInInspector, SerializeField, FormerlySerializedAs("observationStacks")]
```

Comment:
nit: you shouldn't need the FormerlySerializedAs for these; that's only if an existing variable in a serialized object got renamed.